docs/data/data-grid/row-grouping/RowGroupingReordering.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  useGridApiRef,
+  useKeepGroupedColumnsHidden,
+} from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+export default function RowGroupingReordering() {
+  const data = useMovieData();
+  const apiRef = useGridApiRef();
+
+  const initialState = useKeepGroupedColumnsHidden({
+    apiRef,
+    initialState: {
+      rowGrouping: {
+        model: ['company'],
```

Comment:
```suggestion
        model: ['company', 'director'],
```
to better showcase rules when grouped by multiple columns